Source/ThirdParty/skia/CMakeLists.txt

```
@@ -895,9 +901,25 @@ add_library(Skia STATIC
     modules/skcms/src/skcms_TransformSkx.cc
 )
 
+if (is_android)
```

Check `ANDROID` directly here as well:

```suggestion
if (ANDROID)
```

---

Fixed